### ISSUE TITLE
"No hope" mapgen json-error

### ISSUE THREAD

**Issue by LinKennt** [cataclysmbnteam/Cataclysm-BN#5745]
### Describe the bug

Internal mod "No Hope [no_hope]" calls json-error "row X column Y: 'Z' has no terrian, furniture, or other defenition" at the "mapgen weights" Finalizing stage house_inner_garden r13/c15-16 and 18-19, r14/c13-19, r15/c14-19; and s_gas.json r19/c5.

### Steps To Reproduce

1. Create new world with "No Hope" mod
2. Open World

### Screenshots

https://github.com/user-attachments/assets/ce8dd676-facf-46b5-8786-0990d120dfb1

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.26100.1742 (24H2)
- Game Version: aaf5b02 [64-bit]
- Graphics Version: Tiles
- LAPI Version: 2
- Game Language: English [en_US]
- Mods loaded: [
    Bright Nights [bn],
    Disable NPC Needs [no_npc_food],
    Simplified Nutrition [novitamins],
    No Rail Stations [No_Rail_Stations],
    Prevent Zombie Revivication [no_reviving_zombies],
    Limit Fungal Growth [limit_fungal_growth],
    Item Category Overhaul [Item_Category_Overhaul],
    UDP Redux [udp_redux],
    No Hope [no_hope]
]

### Additional context

[debug.log](https://github.com/user-attachments/files/17837718/debug.log)

**Comment by blot-runner** [comment]
Also encountered it, to my knowledge the culprit is palettes.json in the No Hope mod. Can be fixed by adding to the mod palette the symbols used in the vanilla locations, but absent in the mod ones. Namely:

1. Adding the line
       "s̱": "t_region_soil",
to the "terrain" part of the "house_w_foundation_palette" palette,

2. Adding the line
"nested": { "o": { "chunks": [ [ "null", 95 ], [ "house_place_air_conditioner", 5 ] ] } } 
to the same palette and 

3. Adding the line
"M": { "item_group": "vending_medicine", "reinforced": true }
to the "vendingmachines" part of the "road" palette in the palettes.json in the No Hope mod

solved the problem for me. Attaching the corrected file for reference.

[palettes.json](https://github.com/user-attachments/files/17839132/palettes.json)

**Comment by RoyalFox2140** [comment]
We don't support or fix No Hope, but anyone is welcome to step up to maintain No Hope.

**Comment by chaosvolt** [comment]
Nah, if you can help that'd be appreciated. It's more that right now No Hope doesn't have a dedicated maintainer. Part of the problem is the mod is an absolute rat's nest of overrides that can be a pain to sift through,especially since you then have to triage whether any given override is even nessecary in BN (a huge chunk of the mod is dedicated to re-implementing random stuff DDA has obsoleted for "muh realisms" reasons over the years, and a lot of that has at various points been re-added to BN).

**Comment by RoyalFox2140** [comment]
> > We don't support or fix No Hope, but anyone is welcome to step up to maintain No Hope.
> 
> Hmm... does it mean that if I finally learn the black magic of git and choose to fix this WITHOUT becoming a full-time maintainer of the mod, the PR won't be merged?)

We'll merge anybody's work. What I meant was nobody is expected to fix No Hope so if a PR breaks No Hope we don't even know because we stopped running tests on No Hope. It is not anyone's responsibility but anyone can work on it.
